### ISSUE TITLE
What Repository is This?

### ISSUE THREAD

**Issue by ghost** [mecha-cms/0#1]
Hi @Tovic,

I see here that it is a new repository. Is it Mecha v1.2.8 or a new version of the CMS?

Thanks in advance.

**Comment by taufik-nurrohman** [comment]
I just don&rsquo;t use the _branch_ feature because the issue page is just useful enough as a forum. So I created this repository for the first Mecha CMS version in the hope that users will ask something related to Mecha `v1.x.x` here. The goal is:

 - `mecha-cms/mecha` &rarr; used for the stable release
 - `mecha-cms/genome` &rarr; used a the development repository for the _genome_ version
 - `mecha-cms/0` &rarr; used as the development repository for the _0_ version if people still prefer to use that obsolete version (focused only to fix small bugs without adding/removing the current features)

Keep in mind that by creating different version &ldquo;nick-name&rdquo; doesn&rsquo;t mean that I will create another version of Mecha next time. The _genome_ version was released to overcome some bad practice and performance problem in the previous version that can&rsquo;t be fixed away unless I change the whole API. So, if the _genome_ version is stable enough for a long time, then I will not make any future version plans.
